lib_lua/src/luaconf.h

```diff
@@ -527,7 +530,7 @@
 #if defined(LUA_NUMBER_INTEGER)
 #define lua_number2str(s,n)     modp_itoa10((n),s)
 #else
-#define lua_number2str(s,n)     modp_ftoa((n),s, LUA_NUMBER_PRECISION)
+#define lua_number2str(s,n)     modp_dtoa((n),s, LUA_NUMBER_PRECISION)
```

Comment:
Err.. by stdlib I mean the methods in string.h that will convert a double to a char array.